ui/src/containers/projects.create/index.tsx

```diff
@@ -20,6 +20,10 @@ import { CustomFieldName } from "~/components/CustomField";
 import { SheetFooter } from "~/components/sheet/footer";
 import { SheetHeader } from "~/components/sheet/header";
 
+// Setting this to 1000 initially till APIs support filters and sorting.
```

Comment:
Let's rephrase the comment as a `TODO` so that we can find it easily later.
Same for the one above.